<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Service_Create_Description</class>
 <widget class="QDialog" name="Service_Create_Description">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registering a Service Order</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/images.qrc">
    <normaloff>:/emoticons/service.png</normaloff>:/emoticons/service.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="line_Short_Description">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>70</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="placeholderText">
      <string>Short Service Description...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QLabel" name="lbl_Emoticon">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4" colspan="5">
    <widget class="QTextEdit" name="txt_Full_Description">
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Full service description. Max. 1000 chars.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
